components/announcement_bar/payment_announcement_bar/payment_announcement_bar.tsx

```diff
@@ -93,4 +93,4 @@ class PaymentAnnouncementBar extends React.PureComponent<Props> {
     }
 }
 
-export default withGetCloudSubscription(PaymentAnnouncementBar);
```

Comment:
I think you should remove the import statement for `withGetCloudSubscription` too.